### PR TITLE
ICU-21051 Add copyright Remove License from sh

### PR DESCRIPTION
**Pull request by FrankYFTang**
##### Checklist

- [X] Issue filed: https://unicode-org.atlassian.net/browse/ICU-21051
- [X] Updated PR title and link in previous line to include Issue number
- [X] Issue accepted
- [X] Tests included
- [X] Documentation is changed or added

### CODE REVIEW

File: icu4c/source/test/perf/localecanperf/runTest.sh [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
+# Copyright (C) 2020 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html#License
```
Comment (markusicu) [inline icu4c/source/test/perf/localecanperf/runTest.sh:3]: Please remove the `#License` suffix.

(See also ICU-21257 which I created half an hour ago.)

Comment (markusicu) [reply icu4c/source/test/perf/localecanperf/runTest.sh]: Do *not* remove the whole "License & terms" line!

Do remove *only* the `#License` fragment/suffix.

`# License & terms of use: http://www.unicode.org/copyright.html`

Comment (markusicu) [reply icu4c/source/test/perf/localecanperf/runTest.sh]: https://en.wikipedia.org/wiki/URI_fragment

Comment (FrankYFTang) [reply icu4c/source/test/perf/localecanperf/runTest.sh]: Please take another look.

**Comment by FrankYFTang** [comment]
> lgtm tnx please squash

done

**Comment by markusicu** [comment]
The AppVeyor build failure looks unrelated.

**Comment by sffc** [comment]
I'm going to take the liberty to merge this PR since it is blocking https://github.com/icu-units/icu/pull/74.

**Pull request merged**